pkg/util/generate_secret_test.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	"testing"
+)
+
+func TestSecureStringGenerator(t *testing.T) {
+	secretSize := 20
+	uniqueSecrets := make(map[string]bool)
+	for count := 0; count < 5; count++ {
+		secretKey, err := GenerateSecureString(secretSize)
+		if len(secretKey) != 20 {
+			t.Errorf("The given secret is not of length %v", secretSize)
+		}
+		if err == nil {
+			uniqueSecrets[secretKey] = true
+		}
+	}
+	want := 5
+	got := len(uniqueSecrets)
+
+	if want != got {
+		t.Error("The secrets generated by GenerateSecureString() arent unique")
```

Comment:
```suggestion
		t.Errorf("The secrets generated by GenerateSecureString() arent unique, got %d unique secrets", len(uniqueSecrets))
```

Reply:
But actually....

if l := len(uniqueSecrets); l != 5 {
    t.Errorf("The secrets generated by GenerateSecureString() arent unique, got %d unique secrets", l)
}

Reply:
made the above change